lib/modules/device/DevicesController.ts

```diff
@@ -443,4 +463,40 @@ export class DevicesController {
       payloadUuids
     );
   }
+
+  async export(request: KuzzleRequest) {
+    const engineId = request.getString("engineId");
+
+    if (
+      request.context.connection.protocol === "http" &&
+      request.context.connection.misc.verb === "GET"
+    ) {
```

Comment:
Just asking, but why do we keep this controller with radically changing behavior (with no shared behavior) according to the type of connection ? Why don't we use two controllers ?

Reply:
I'm not sure, but I think we need to improve core to can split this.
However, this behavior is due to the fact that it takes 2 type of request for an export.
First, we need to prepare export with `POST` request, then retrieve file with a `GET` request.